### ISSUE TITLE
Pagination button link labels are not descriptive on PLP

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#164]
Pagination buttons incorrectly say "Current page X" as the spoken label, where "X" is the page number, e.g. if I want to load results for page 3, the spoken button label is "current page 3". This is not an accurate description of what that button does. The label should be:

( 1 ) for current page: "Current page 1" (This is the same as current, keep this)
( 2 ) "Page 2 display results"
( 3 ) "Page 3 display results"

NOTE: I know this goes against guideline 2.5.3, Label in Name, but I can't think of a descriptive label that would start with the page number itself (the visible label). I am willing to bend the guideline for the purpose of clarity by proposing we pre-pend the word "page".

**Comment by brianelton** [comment]
Fixed on public site
